flowsa/methods/flowbysectormethods/CAP_HAP_national_2017.yaml

```diff
@@ -196,9 +196,8 @@ source_names:
     FBS_datapull_fxn: stewicombo_to_sector
     inventory_dict: {"NEI":"2017", "TRI":"2017"}
     local_inventory_name: 'NEI_TRI_air_2017'
+    clean_fbs_df_fxn: !script_function:EPA_NEI drop_GHGs
```

Comment:
`clean_fbs_df_fxn` didn't exist at the time of this method. I believe this change should be fine, but I'd like to confirm that keeping the GHGs until later (i.e. after `prepare_stewi_fbs` doesn't cause any issues).

Reply:
Agreed.